ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2909,3 +2909,40 @@ def test_numpy_instance_mod__(
         method_flags=method_flags,
         on_device=on_device,
     )
+
+#ptp    
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="ptp",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_axis=-1,
+        max_axis=0,
+        force_int_axis=True,
+    ),
```

Comment:
Also, I think `axis` can be a tuple as well, so no need to force it to be an integer. And it could take any value inside the array's bounds, not only -1 and 0.